back/engines/commercial/analysis/app/controllers/analysis/web_api/v1/analyses_controller.rb

```diff
@@ -30,7 +30,12 @@ def show
           render json: WebApi::V1::AnalysisSerializer.new(
             @analysis,
             params: jsonapi_serializer_params,
-            include: serializer_includes
+            include: [
+              *serializer_includes,
+              :'main_custom_field.matrix_statements',
+              :'additional_custom_fields.matrix_statements',
+              :'all_custom_fields.matrix_statements'
+            ]
```

Comment:
The fix to the matrix statement bug: these need to be included, because the 'backup endpoint' was not properly implemented

Reply:
So this is not directly related to the PoC